core/common/src/main/java/alluxio/worker/block/BlockWorker.java

```diff
@@ -115,6 +115,26 @@ void commitBlock(long sessionId, long blockId, boolean pinOnCreate)
   String createBlock(long sessionId, long blockId, int tier, String medium, long initialBytes)
```

Comment:
Can this be removed? since we have the new API? or you prefer to do it in another PR?

Reply:
That could be impacting on other code path, introduce a lot of change. It is expected to do in other PR given the constraint.

Reply:
Removing the old createBlock API will be done in another PR